pkg/engine/templates_generated.go

```diff
@@ -37045,6 +36995,22 @@ MASTER_CONTAINER_ADDONS_PLACEHOLDER
     {{WrapAsVariable "environmentJSON"}}
 {{end}}
 
+disk_setup:
+  /dev/disk/azure/scsi1/lun0:
+    table_type: gpt
+    layout: true
+    overwrite: false
+fs_setup:
+  - label: etcd_disk
+    filesystem: ext4
+    device: /dev/disk/azure/scsi1/lun0
+    extra_opts:
+      - -F
+      - -E
+      - lazy_itable_init=1,lazy_journal_init=1
```

Comment:
Any particular reason why AKS-Engine eneds these extra options? 
http://manpages.ubuntu.com/manpages/bionic/man8/mke2fs.8.html

Also, do you think adding `-e error-behavior` would be beneficial? Just floating ideas and aks-engine may want to either continue normal execution or cause a kernel panic/stop completely if etcd data disk fs setup fails.

Also, you may be interested in the `-U` option, which allows you to specify a UUID to the single partition within the etcd data disk.

This allows the etcd datadisks's single partition to be visible within `/dev/disk/by-uuid/`, so if you specified `-U 11111111-1111-1111-1111-111111111111`, then `/dev/disk/by-uuid/11111111-1111-1111-1111-111111111111` would be a symlink to partition 1 (the single partition) within the etcd data disk.

Just floating ideas that the aks-engine team may find beneficial, so feel free to just cherry pick the ones that suit your needs.